Attributes/ValidationAttributes/MaxFileSizeAttribute.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace GovUkDesignSystem.Attributes.ValidationAttributes
+{
+    [AttributeUsage(AttributeTargets.Field | AttributeTargets.Property, AllowMultiple = false, Inherited = true)]
+    public class MaxFileSizeAttribute : ValidationAttribute
+    {
+        private readonly int _maxFileSize;
+        public MaxFileSizeAttribute(int maxFileSize)
+        {
+            _maxFileSize = maxFileSize;
+        }
+
+        public override bool IsValid(object value)
+        {
+            var file = value as IFormFile;
+            if (file == null)
+            {
+                return false;
```

Comment:
If there's no file I don't think it should fail a max size test. If we want to make the file required then we should return the relevant error message, or the user can use the GovUkValidateRequired attribute.